script/entrypoint

```diff
@@ -18,11 +18,14 @@ if [ -n "$GOOGLE_APPLICATION_CREDENTIALS" ] && which gcloud > /dev/null; then
 fi
 
 if [ "$#" = 0 ]; then
-    # Default to 20 workers because most of the tests are io-bound
-    exec pytest --black --flake8 --mypy-ignore-missing-imports --pydocstyle --numprocesses 20
+    # Default to 8 workers because most of the tests are io-bound
+    # When using pytest-xdist, total worker initialization time scales
+    # linearly to the number of workers. 8 workers seems to bring the best
+    # performance. See https://github.com/pytest-dev/pytest-xdist/issues/346
+    exec pytest --black --flake8 --mypy-ignore-missing-imports --pydocstyle -n 8
```

Comment:
The linked issue explains why having a lot of processes doesn't necessarily mean that tests are running faster. With 20 processes the initialization took ~5 minutes, with 8 it's much faster.